src/main/java/com/onevizion/scmdb/dao/DbScriptDaoOra.java

```diff
@@ -15,18 +17,17 @@
 import java.sql.DatabaseMetaData;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 
 @Repository
 public class DbScriptDaoOra extends AbstractDaoOra {
 
     private static final String UPDATE = "update db_script set file_hash = :fileHash,text = :text,ts = :ts where db_script_id = :id";
     private static final String CREATE = "insert into db_script (name,file_hash,text,ts,output,type,status) values (:name,:fileHash,:text,:ts,:output,:type.id,:status.id)";
     private static final String DELETE = "delete from db_script where db_script_id = ?";
-    private static final String READ_ALL = "select * from db_script";
+    private static final String READ_ALL = "select db_script_id,name,file_hash,null as text,ts,type,status from db_script";
+    private static final String READ_ALL_BY_ID = "select db_script_id,name,file_hash,text,ts,type,status from db_script where db_script_id in (:ids)";
```

Comment:
name is confusing, read all or read by id?

Reply:
renamed